src/css/main.css

```diff
@@ -587,8 +587,8 @@ span.subcomponent {
   cursor: grabbing;
 }
 
-code, pre {
```

Comment:
@cvan So have just `code, pre, input... { font-family: 'Roboto Mono', ...}` and this too: `.wf-active { font-family: 'Roboto Mono', ... }`?